modules/queue/unique_queue.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package queue
+
+import (
+	"context"
+	"fmt"
+)
+
+// UniqueQueue defines an interface to save an issue indexer queue
```

Comment:
the comment is not correct.

Reply:
I think the comments might not be correct in the original too.

Reply:
Done.